api/api.go

```diff
@@ -1000,6 +1000,10 @@ func (r *request) toHTTP() (*http.Request, error) {
 		return nil, err
 	}
 
+	if strings.HasPrefix(r.url.Host, "/") {
```

Comment:
Suggest adding a comment here that mentions the socket problem. Out of context, I think this hack is going to be non-obvious.

Reply:
add comment in 4452224d6ac786d1d1a723490d08aa7038088f31